src/components/App/index.spec.tsx

```diff
@@ -77,6 +77,17 @@ describe(__filename, () => {
     expect(root.find(Navbar)).toHaveLength(0);
   });
 
+  it('does not dispatch setAuthToken on mount when authToken is already present in the state', () => {
+    const authToken = 'my-token';
+    const store = configureStore();
+    store.dispatch(apiActions.setAuthToken({ authToken }));
+    const dispatch = spyOn(store, 'dispatch');
+
+    render({ authToken, store });
+
+    expect(dispatch).not.toHaveBeenCalled();
+  });
+
   it('dispatches setAuthToken on mount when authToken is valid', () => {
```

Comment:
You might want to rename this test to indicate `when authToken is not present in state`, or something similar.